packages/ui/src/council/components/election/CandidateVote/CandidateVote.tsx

```diff
@@ -76,7 +79,9 @@ export const CandidateVote = ({
           <StakeAndVotesRow>
             {hasOwnStake && (
               <>
-                <Subscription>My Stake</Subscription>
+                <Tooltip tooltipText={tooltipLabel}>
```

Comment:
add `tooltipLinkURL="https://joystream.gitbook.io/testnet-workspace/system/council#candidacy"`

Reply:
yes i will add it